addons/static/templates/policy_cache_transformer_persona.json

```diff
@@ -1016,76 +989,185 @@
         "end-one-entity:*",
 
         "end-two-entity-type:AIApplication",
-        "end-two-entity-type:AIModel",
         "end-two-entity-classification:*",
         "end-two-entity:{entity}"
       ],
-      "actions": ["update-relationship"]
+      "actions": ["remove-relationship"]
+    }
+  ],
+  "persona-ai-application-add-classification": [
+    {
+      "policyResourceCategory": "ENTITY",
+      "policyType": "ACCESS",
+      "resources": [
+        "entity:{entity}",
+        "entity-type:AIApplication",
+        "entity-classification:*",
+        "classification:*"
+      ],
+      "actions": [
+        "entity-add-classification",
+        "entity-update-classification"
+      ]
+    }
+  ],
+  "persona-ai-application-remove-classification": [
+    {
+      "policyResourceCategory": "ENTITY",
+      "policyType": "ACCESS",
+      "resources": [
+        "entity:{entity}",
+        "entity-type:AIApplication",
+        "entity-classification:*",
+        "classification:*"
+      ],
+      "actions": [
+        "entity-update-classification",
+        "entity-remove-classification"
+      ]
+    }
+  ],
+  "persona-ai-model-read": [
+    {
+      "policyType": "ACCESS",
+      "policyResourceCategory": "ENTITY",
+      "resources": [
+        "entity:{entity}",
+        "entity-type:AIModel",
+        "entity-classification:*"
+      ],
+      "actions": ["entity-read"]
+    }
+  ],
+  "persona-ai-model-create": [
+    {
+      "policyResourceCategory": "ENTITY",
+      "policyType": "ACCESS",
+      "resources": [
+        "entity:*",
+        "entity:default/ai/aimodel/*",
+        "entity-type:AIModel",
+        "entity-classification:*"
+      ],
+      "actions": ["entity-create", "entity-update"]
     }
   ],
-  "persona-aiasset-remove-terms": [
+  "persona-ai-model-update": [
+    {
+      "policyResourceCategory": "ENTITY",
+      "policyType": "ACCESS",
+      "resources": [
+        "entity:{entity}",
+        "entity:default/ai/aimodel/*",
+        "entity-type:AIModel",
+        "entity-classification:*"
+      ],
+      "actions": ["entity-update"]
+    },
     {
       "policyResourceCategory": "RELATIONSHIP",
       "policyType": "ACCESS",
-      "description": "Link/unlink term to this AI asset",
+      "description": "Link/unlink AIModel to any AIApplication",
       "resources": [
         "relationship-type:*",
-
-        "end-one-entity-type:AtlasGlossaryTerm",
+        "end-one-entity-type:AIModel",
         "end-one-entity-classification:*",
         "end-one-entity:*",
 
         "end-two-entity-type:AIApplication",
-        "end-two-entity-type:AIModel",
         "end-two-entity-classification:*",
         "end-two-entity:{entity}"
```

Comment:
Since this is model-update policy, it should be `end-two-entity:*` & `end-one-entity:{entity}`